wolfcrypt/src/sp_int.c

```diff
@@ -11631,6 +11948,8 @@ static int _sp_sqr_4(sp_int* a, sp_int* r)
 }
 #else /* SQR_MUL_ASM */
 /* Square a and store in r. r = a * a
+ *
+ * Comba implementation.
```

Comment:
This `_sp_sqr_4` function has a reference for `to` that doesn't exist. I can't seem to locate the build combination to trigger the error. All my testing passed fine on an STM32L4.

See `sp_int.c:11967`
```
#if defined(WOLFSSL_SP_ARM_THUMB) && SP_WORD_SIZE == 32
    to = 0;
#endif
```

Reply:
Fixed